### ISSUE TITLE
Can't use 24 column grid

### ISSUE THREAD

**Issue by buesing** [JSxMachina/react-grid-system#56]
I'm trying to set up a grid system with 24 columns, and running into problems when trying to make a `<Col>` wider than 12 Columns. The problem seems to be in `src/utils.js` on line 38: 
```
export const normalizeColumnWidth = width => Math.max(0, Math.min(12, width));
```
Now with a `gridColumns` value of 24, my column that is supposed to be 18 wide is capped to 12 columns. Shouldn't this value be capped to the gridColumns value from the context and not always 12?

**Comment by gerbenmeyer** [comment]
Fair point, I'll make a fix for that.

**Comment by gerbenmeyer** [comment]
Should be fixed now in the latest version (3.2.3).
